static/app/components/onboardingWizard/content.tsx

```diff
@@ -651,6 +646,8 @@ const TaskGroupWrapper = styled('div')`
   border-radius: ${p => p.theme.borderRadius};
   padding: ${space(1)};
 
+  background-color: ${p => p.theme.background};
```

Comment:
This doesn't have an effect on the old sidebar panel since the background is already this color